src/main/java/loose/oose/fis/lab/student/manager/HelloApplication.java

```diff
@@ -0,0 +1,24 @@
+package loose.oose.fis.lab.student.manager;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+
+import java.io.IOException;
+
+public class HelloApplication extends Application {
+    @Override
+    public void start(Stage primaryStage) throws Exception {
+        Parent root = FXMLLoader.load(getClass().getResource("fxml/view-students.fxml"));
+        primaryStage.setTitle("Student Manager");
```

Comment:
change title to FIS-Student manager pls 
```suggestion
        primaryStage.setTitle("Student Manager");
```